doc/nrf/ug_radio_fem.rst

```diff
@@ -127,12 +129,14 @@ This feature uses a model to compensate the FEM gain for the following external
 The model assumes that the FEM supply voltage is constant.
 To provide the value of this voltage to the MPSL subsystem, use the :kconfig:option:`CONFIG_MPSL_FEM_POWER_VOLTAGE` option.
```

Comment:
This starts to be a problem, if the model is enabled by default - the user might add nRF21540 FEM with SPI into devicetree, but not change this Kconfig - maybe instead of this Kconfig option this should be done in device tree? I would say this is a hardware parameter.

Reply:
That is a valid concern and the solution you're suggesting is reasonable. I'll start working on the implementation, changes in this PR will follow when there's something to document.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/9073 introduces related changes.

Reply:
So is line 134 still valid?

Reply:
No, it's not. Thanks for catching that, I'll remove it.